docs/PID tuning.md

```diff
@@ -48,6 +48,10 @@ and earlier.
 One of the quirks with this controller is that if you increase the P value for an axis, the maximum rotation rates for
 that axis are lowered. Hence you need to crank up the pitch or roll rates if you have higher and higher P values.
 
+The Roll/Pitch and Yaw rate settings do not work like TX stick scaling. Instead they cause a proportional reduction in PID values with increased YPR (Yaw Pitch Roll) axis stick deflections. eg) A Pitch/Roll Rate setting of 0.5 will reduce the three PID terms for Pitch & Roll to 50% at full stick deflection. Warning! A Pitch/Roll Rate or Yaw Rate setting of 1.0 will reduce all of the PID terms to zero at full stick deflection!
+
```

Comment:
(deleted comments that were wrong)

ACC term (leveling):
- P is `2 * rcCommand` scaled with `pidProfile->P8[PIDLEVEL]` and limited to +-`pidProfile->D8[PIDLEVEL]`
- I term is integrated `2 * rcCommand` scaled with 'pidProfile->I8[PIDLEVEL]'.

GYRO:
- I term error is computed from gyroData and rcCommand using 'pidProfile->P8[axis]' (not dynP8), then scaled with pidProfile->I8[axis]
- P term is rcCommand directly

ACC and GYRO is linearly mixed according to deflection in horizon mode.

`gyroData[axis] / 4) * dynP8[axis]` is subtracted from P-term

D term is computed only from filtered `gyroData[axis]` (no `rcCommand` influence) and scaled with `dynD8[axis]`

motor command is `P+I-D`

So `rate` decreases only influence of gyro data, down to zero influence with rate = 1 and full deflection, but signal from stick position is not affected.

This code is IMO terrible hack ...